### PR TITLE
Plural form of some of the options in Cron Job, Job, Gateway, Oath2 Client, and DNE Entry views

### LINKED ISSUE #817: Components that allow for choosing/typing more than one option should be in plural form
**Description**
Not all components that allow for choosing/typing more than one option are in plural form all around Busola. It should unified. For example:
Zrzut ekranu 2022-01-17 o 11 18 32
Slectors

**Expected result**
Whenever a component has a possibility to choose/type multiple items, it should be written in plural form.

**Actual result**

**Steps to reproduce**

**Troubleshooting**

### PR DESCRIPTION
**Pull request by grego952**
**Description**

Changes proposed in this pull request:

- add plural form to CRD with multiple choice/type option
- small text refactor

**Related issue(s)**
Resolves #817

### CODE REVIEW

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -1244,7 +1244,7 @@ jobs:
     containers-readonly-in-edit: After a Job is created, the containers are read-only.
     labels:
       args: Args
-      command: Command
+      command: Commands
```
Comment (Wawrzyn321) [inline core/src/i18n/en.yaml:1247]: It's a single command, consisting of multiple parts.
```suggestion
      command: Command
```

File: core/src/i18n/en.yaml [hunk 2]
```diff
@@ -1407,7 +1407,7 @@ persistent-volume-claims:
     match-expressions: Match expressions
     match-labels: Match labels
     operator: Operator
-    selector: Selector
+    selector: Selectors
```
Comment (Wawrzyn321) [inline core/src/i18n/en.yaml:1410]:
```suggestion
    selector: Selector
```
K8s calls it "selector".

File: core/src/i18n/en.yaml [hunk 3]
```diff
@@ -1378,7 +1378,7 @@ oauth2-clients:
     post-logout-redirect-uris: Post-Logout Redirect URIs
     redirect-uris: Redirect URIs
     response-types: Response Types
-    scope: Scope
+    scope: Scopes
```
Comment (Wawrzyn321) [inline core/src/i18n/en.yaml:1381]: I'd go with "scope", for example Luigi calls it like that https://docs.luigi-project.io/docs/authorization-configuration, even though there are multiple 
```suggestion
    scope: Scope
```

Comment (Wawrzyn321) [reply core/src/i18n/en.yaml:1381]: `scope defines the permissions to request at login.`
yeah, it's a single scope with a set of permissions.

Comment (grego952) [reply core/src/i18n/en.yaml:1381]: But here you have scopes - https://docs.apigee.com/api-platform/security/oauth/working-scopes

Comment (Wawrzyn321) [reply core/src/i18n/en.yaml:1381]: Yup, and there's a `scope` in JSON
```
  "scope" : "A",
```
I guess they named the article like that, just like https://www.ocg.nsw.gov.au/child-safe-organisations/working-with-children-check, even though you could also work with one child.

**Review by Wawrzyn321 — APPROVED** [review]

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**